### PR TITLE
Features/hull node upgrade

### PR DESCRIPTION
**Pull request by mkawalek**
* logs follow new log-convention document
* hull-node 11.7 upgrade

### CODE REVIEW

File: server/actions/fetch-all.js [hunk 1]
```diff
@@ -17,7 +17,7 @@ export default function fetchAll(req: Request, res: Response) {
     return res.redirect("back");
   }
 
-  req.hull.client.logger.info("fetchAll.started");
+  req.hull.client.logger.debug("fetchAll.started");
```
Comment (michaloo) [inline server/actions/fetch-all.js]: @mkawalek could we use `incoming.job.*` notation here?

Comment (mkawalek) [inline server/actions/fetch-all.js]: @michaloo Yes, we can use that over here. The question is if this is the only place where we want to use incoming.job.& notation ?

Comment (mkawalek) [inline server/actions/fetch-all.js]: @michaloo I think that we should use this notation also within jobs/fetch.js. Do you agree with me ? :)

Comment (michaloo) [inline server/actions/fetch-all.js]: @mkawalek yes, use it in both places :)

Comment (mkawalek) [inline server/actions/fetch-all.js]: @michaloo I updated PR, could you please take a look at it one more time ? :)

**Pull request merged**